code/parse/parselo.h

```diff
@@ -158,8 +158,8 @@ extern void parse_string_flag_list(int *dest, flag_def_list defs[], size_t defs_
 // A templated version of parse_string_flag_list, to go along with the templated flag_def_list_new.
 // If the "is_special" flag is set, or a string was not found in the def list, it will be added to the unparsed_or_special_strings Vector
 // so that you can process it properly later
-template<class T, class Flagset>
-void parse_string_flag_list(Flagset& dest, flag_def_list_new<T> defs [], size_t n_defs, SCP_vector<SCP_string>* unparsed_or_special_strings)
+template<class Flagdef, class Flagset>
+void parse_string_flag_list(Flagset& dest, const Flagdef defs[], size_t n_defs, SCP_vector<SCP_string>* unparsed_or_special_strings)
```

Comment:
With the change in usage, I assume the last parameter should be `unparsed` rather than `unparsed_or_special`

Reply:
Ah, ok.